proto/process/agent.proto

```diff
@@ -498,6 +498,8 @@ message Pod {
 	bytes yaml = 14;
 	string uid = 15;
 	Host host = 16; // Used post-resolution
+	string kubectlStatus = 17; // pod status displayed by "kubectl get pod"
```

Comment:
Any reason why this shouldn't just be `status`?

Reply:
good point, updating 👍